src/nix/flake-init.md

```diff
@@ -30,7 +30,7 @@ using `-t`.
 # Template definitions
 
 A flake can declare templates through its `templates` output
-attribute. A template has two attributes:
+attribute. A template has three attributes:
```

Comment:
```suggestion
attribute. A template has the following attributes:
```

I recommend not hard-coding this number at all, then there will never maintenance overhead again. (We should have learned that like in the 1960s.)


@Ericson2314 😉